### DISCUSSION TITLE
CTK Circle Loading Widget

### DISCUSSION THREAD

**Discussion by anamite** [TomSchimansky/CustomTkinter#2164] [Showcase]
### Custom Circular/Gauge/Meter Loading widget 

- Simple
- Clean
- Easy to use

![image](https://github.com/TomSchimansky/CustomTkinter/assets/77412636/0adcecfe-d395-4b3f-84d8-9002324d730c)

[Repo](https://github.com/anamite/ctk_widget?tab=readme-ov-file)

**Comment by Akascape** [comment]
@anamite Very good, I also made some circular meter widgets for tkinter called [tkdial](https://github.com/Akascape/TkDial)

I will put your repo in tkinter toolkit: https://github.com/Akascape/tkinter-toolkit

**Comment by Akascape** [comment]
@anamite I also recommend making a loading animation without percentage, means the circular bar we see on a loading screen/video. Tkinter/Customtkiner lacks that type of widget.

**Comment by anamite** [comment]
Hi @Akascape , thanks for adding it to your toolkit :)
Is a really good suggestion.
I will surely try to add this to the package later :)

**Reply by Akascape** [reply]
@anamite Great, let me know when it's ready.
